posthog/tasks/usage_report.py

```diff
@@ -802,37 +802,35 @@ def get_teams_with_recording_bytes_in_period(
     return result
 
 
-@shared_task(**USAGE_REPORT_TASK_KWARGS, max_retries=0)
 def capture_report(
     *,
+    pha_client: PostHogClient,
     capture_event_name: str,
-    org_id: Optional[str] = None,
+    organization_id: Optional[str] = None,
     team_id: Optional[int] = None,
     full_report_dict: dict[str, Any],
     at_date: Optional[datetime] = None,
```

Comment:
should this be a str?

Reply:
Yeah, didn't change anything here so not sure why it's failing. Will fix.